From f6b5d7b6883c986f4efbcd3e28bd1ef8f26946a9 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Mon, 16 Sep 2013 15:17:31 +0800
Subject: [PATCH 251/602] ENGR00279791 CTS case
 testAllCharacterDevicesAreSecure failure

All character device should not have wrx permissions for other
Remove non-used character for imx6

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index 9099a66..657d437 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -1,6 +1,6 @@
 /dev/pmem_gpu             0660   system     graphics
 /dev/ttymxc4              0660   bluetooth  bluetooth
-/dev/snd/*                0664   system     audio
+/dev/snd/*                0660   system     audio
 /dev/ttyUSB*              0640   radio      radio
 /dev/ttyACM*              0640   radio      radio
 /dev/video*               0660   system     camera
@@ -9,12 +9,8 @@
 /dev/video2               0660   system     camera
 /dev/video16              0660   system     graphics
 /dev/mxc_ipu              0660   media      system
-/dev/mxc_iim              0664   system     media
-/dev/mxs_viim             0664   system     media
-/dev/fsl_cache            0666   system     graphics
-/dev/gsl_kmod             0666   system     graphics
 /dev/mxc_vpu              0660   media      system
-/dev/uinput               0666   system     input
+/dev/uinput               0660   system     input
 /dev/ttymxc2              0660   system     gps
 /dev/ttya0                0660   system     gps
 /dev/ptya0                0660   system     gps
-- 
1.8.0

